src/hokusai/hokusai.yml

```diff
@@ -160,9 +170,22 @@ jobs:
           name: Deploy
           command: hokusai staging deploy $CIRCLE_SHA1 --update-config
           no_output_timeout: << parameters.time-out >>
-      - run:
-          name: Update Staging branch
-          command: git push git@github.com:artsy/<< parameters.project-name >>.git $CIRCLE_SHA1:refs/heads/staging --force
+      - git-push-staging-branch-command:
+          project-name: << parameters.project-name >>
+
+  git-push-staging-branch-job:
```

Comment:
Wouldn't `push-staging-branch` be sufficient? This sounds a little verbose.

Reply:
[sure thing.](https://github.com/artsy/orbs/pull/151/commits/5d241adaf2d835b5ae7d802c7ae685e32308ee68)